### DISCUSSION TITLE
accumulation err/fs-security-constraint

### DISCUSSION THREAD

**Discussion by magnusboman** [fluxcd/flux2#2757] [Q&A] [answered]
Hello,

In my `clusters/management/flux-system/kustomization.yaml` I'm trying to include `../../../environments/test/kustomization.yaml`, but get the following error:

```
✗ accumulating resources: accumulation err='accumulating resources from '../../../environments/test/kustomization.yaml': fs-security-constraint read /tmp/flux-bootstrap-334132595/environments/test/kustomization.yaml: path '/tmp/flux-bootstrap-334132595/environments/test/kustomization.yaml' is not in or below '/tmp/flux-bootstrap-334132595/clusters/management'': fs-security-constraint abs /tmp/flux-bootstrap-334132595/environments/test/kustomization.yaml: path '/tmp/flux-bootstrap-334132595/environments/test/kustomization.yaml' is not in or below '/tmp/flux-bootstrap-334132595/clusters/management'
```

All the files are in the same git repository so they are in fact "in or below" the `clusters/management` directory.

Anyone know how to get this to work, as I rather have my apps/environments etc in the root folder of the git repo.

**Comment by stefanprodan** [comment]
Can you post here the kustomization.yaml please

**Comment by magnusboman** [comment]
Hi @stefanprodan , here's my kustomization file:

$ cat clusters/management/flux-system/kustomization.yaml 
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
- gotk-components.yaml
- gotk-sync.yaml
- ../../../environments/test/kustomization.yaml
```

$ cat environments/test/kustomization.yaml 
````
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
- ../../base/tf-controller.yaml

**Reply by stefanprodan** [reply]
You can't add a `kustomization.yaml` to the resources, it has to be the dir so `../../../environments/test`.

**Comment by stefanprodan** [comment, marked as answer]
This bug has been fixed in https://github.com/fluxcd/flux2/pull/2764 and it will be included in the next Flux release.
